app/models/group.rb

```diff
@@ -3,6 +3,7 @@
 class Group < ApplicationRecord
   has_many :group_memberships, dependent: :destroy
   has_many :users, through: :group_memberships
+  has_many :invites, dependent: :destroy
```

Comment:
is Rails really smart enough to remove invites where `recipient` and `sender` are set to this user? Specs Or It Didn't Happpen!